charts/app-config/values-integration.yaml

```diff
@@ -1180,6 +1185,11 @@ govukApplications:
       extraEnv:
         - name: PLEK_HOSTNAME_PREFIX
           value: draft-
+        - name: PUBLISHING_API_BEARER_TOKEN
```

Comment:
Are you sure the draft deployment needs this? Is there a use case where you need to run the rake task from the draft version? No problem if so, but if not, please remove the unneeded secretKeyRefs from the draft deployments.

Reply:
Thanks, I've updated the comment to instead make sure they exist. The comment is exactly about what you said - to check if the secrets have been configured in SIgnon.

I am not sure about the second comment, could you explain a bit about how draft works (I joined reasonably recently)? If it's not enabled in draft, how would updates propagate there? Is there another mechanism than running a rake task there?

Reply:
Do you need to run the rake task on the -draft version? Or just the regular version?

Reply:
I am not sure how draft works. If publishing api is shared between draft and non-draft, I don't think that'd be necessary, as the requests would be exactly the same.

Reply:
It literally just comes down to whether you would run the Rake task on the draft version or just the regular version.